nuget.config

```diff
@@ -4,5 +4,6 @@
     <add key="SteeltoeDev" value="https://www.myget.org/F/steeltoedev/api/v3/index.json" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
     <add key="DotnetCore" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+    <add key="OpenTelemetryDev" value="https://www.myget.org/F/opentelemetry/api/v3/index.json" /> 
```

Comment:
Is there substantial difference between the package available on nuget.org and what's available here? If we ship with a dependency from MyGet, others will need to add this to their nuget.config until we change to a version that's on nuget.org. If we need to have this, it should be clearly documented and have a follow-up issue logged somewhere

Reply:
I changed it to nuget version